cypress/utils/utils.ts

```diff
@@ -272,7 +272,7 @@ export function notExists(value: string, tableSelector = commonView.appTable): v
     cy.get(tableSelector).then(($tbody) => {
         if ($tbody.text() !== "No data available") {
             selectItemsPerPage(100);
-            cy.get(tableSelector, { timeout: 5 * SEC }).should("not.contain", value);
+            cy.get(tableSelector, { timeout: 120 * SEC }).should("not.contain", value);
```

Comment:
**suggestion:** Maybe the timeout should be a parameter with a default value. 

Probably we don't want every test that uses this method to wait 2 minutes before failing

Reply:
I'll leave notExists() as is . The test is passing with the other changes. 
The value of SEC could be increased if there is a need in the future.